hazelcast/src/main/java/com/hazelcast/jet/impl/connector/ReadJdbcP.java

```diff
@@ -191,4 +204,34 @@ private static Exception close(AutoCloseable closeable) {
         }
         return null;
     }
+
+    private static void setAutoCommitIfNecessary(Connection connection, Properties properties) {
+        try {
+            String key = JdbcPropertyKeys.AUTO_COMMIT;
+            if (properties.containsKey(key)) {
+                String value = properties.getProperty(key);
+
+                if (isBoolean(value)) {
+                    boolean autoCommit = Boolean.parseBoolean(value);
+                    connection.setAutoCommit(autoCommit);
+                }
+            }
+        } catch (SQLException exception) {
+            LOGGER.severe("Error setting setAutoCommit to PreparedStatement", exception);
+        }
+    }
+
+    private static void setFetchSizeIfNecessary(PreparedStatement statement, Properties properties) {
+        try {
+            String key = JdbcPropertyKeys.FETCH_SIZE;
+            if (properties.containsKey(key)) {
+                int fetchSize = Integer.parseInt(properties.getProperty(key));
```

Comment:
For boolean we have this `if (isBoolean()` check, while here we don't have `if (isInteger()`.